### PR TITLE
[add] account_due_list

### PR DESCRIPTION
**Pull request by scigghia**

### CODE REVIEW

File: account_due_list/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    'version': '0.1',
+    'category': 'Generic Modules/Payment',
+    'description': """
+A due list of pending payments. The list contains every expected payment,
+generated by invoices. The list is fully filterable.
+    """,
+    'author': ['Agile Business Group, Zikzakmedia SL'],
+    'website': 'http://www.agilebg.com',
+    'license': 'AGPL-3',
+    "depends": [
```
Comment (LeartS) [inline account_due_list/__openerp__.py:38]: I'm ok with both single and double quotes, but choose one! :)

File: account_due_list/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                    10
+                ),
+            }),
+    }
+
+    def fields_view_get(
+        self, cr, uid, view_id=None, view_type='form',
+        context=None, toolbar=False, submenu=False
+    ):
+        model_data_obj = self.pool['ir.model.data']
```
Comment (LeartS) [inline account_due_list/account_move_line.py:140]: Didn't you use `_pool` before?

**Comment by LeartS** [comment]
Some nitpicking. I don't know what the module is supposed to do to really review it.

**Comment by scigghia** [comment]
Hi @LeartS i've ported this module in few minutes, because my main effort is on ricevute bancarie that depends on account_due_list

**Comment by eLBati** [comment]
The diff between 7.0 and 8.0: https://gist.github.com/eLBati/85dda5c70961497917ed

:+1:

**Comment by hurrinico** [comment]
According with @LeartS for the fix, from a functional point of view :+1: for me

File: account_due_list/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+##############################################################################
+{
+    'name': "Payments Due list",
+    'version': '0.1',
+    'category': 'Generic Modules/Payment',
+    'description': """
+A due list of pending payments. The list contains every expected payment,
+generated by invoices. The list is fully filterable.
+    """,
+    'author': ['Agile Business Group, Zikzakmedia SL'],
```
Comment (pedrobaeza) [inline account_due_list/__openerp__.py:35]: Remove [] from author.

**Comment by omar7r** [comment]
:+1:

File: account_due_list/account_move_line.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+            inv_ids = invoice_pool.search(
+                cr,
+                uid,
+                [('move_id', '=', line.move_id.id)],
+                context=context
+            )
+            if len(inv_ids) > 1:
+                raise orm.except_orm(
+                    _('Error'),
+                    _('Incongruent data: move %s has more than one invoice')
```
Comment (gurneyalex) [inline account_due_list/account_move_line.py:47]: s/Incongruent/Inconsistent/

**Comment by elacunza** [comment]
Is there anything I can do to further move this PR?

**Comment by eLBati** [comment]
Hello @elacunza you can make a PR to `scigghia:8.0-account_due_list` (or directly to `OCA:8.0` if scigghia has no time to review it) fixing
https://github.com/OCA/bank-payment/pull/84#discussion_r27281505
and
https://github.com/OCA/bank-payment/pull/84#discussion_r28784929
Thanks

**Comment by elacunza** [comment]
Just added a new PR with the issues raised fixed, thanks.

**Comment by eLBati** [comment]
As target of https://github.com/OCA/bank-payment/pull/164 is `OCA:8.0`, I'm closing this

**Pull request closed without merging**